docs/instruction/01-store.md

```diff
@@ -35,7 +35,7 @@ nohup sh bin/mqbroker -n localhost:9876 &
 tail -f ~/logs/rocketmqlogs/broker.log
 ```
 
-The deployment of eventmesh-store has finished, please go to the next step: [Start Eventmesh-Runtime](02-runtime.md)
+The deployment of eventmesh-store has finished, please go to the next step: [Start Eventmesh-Runtime](https://eventmesh.apache.org/docs/instruction/runtime/)
```

Comment:
I'd recommend using relative links so that you can open the markdown files locally and in the github repository, rather than jumping to a browser.